packages/node-sdk/src/client.ts

```diff
@@ -647,6 +647,45 @@ export class BucketClient {
       targetingVersion: feature?.targetingVersion,
     });
   }
+
+  /**
+   * Gets evaluated features with the usage of remote context.
+   * This method triggers a network request every time it's called.
+   *
+   * @param additionalContext
+   * @returns evaluated features
+   */
+  public async getFeaturesRemote(
```

Comment:
wonder if we can make `getFeatures` work in both cases based on a boolean. Typescript should allow us to change the return value to either promise or not based on the types of args. Or maybe that's too confusing

Reply:
That probably won't work because one is async and the other one not.